text/0000-struct-sensor-id.md

```diff
@@ -0,0 +1,30 @@
+# Struct sensor_id
+
+## Summary
+[summary]: #summary
+
+Replace the string sensor_id by a structure with an enum sensor type and a string.
```

Comment:
The way I think about this is to add the possibility to attach sensor metadata to a the sensor_id.

Reply:
Done